pkg/kubeconfig/kubeconfig.go

```diff
@@ -72,6 +75,41 @@ func newConfigLoader(existing *cluster.Cluster) (*configLoader, error) {
 		sshconfig: sshConfig,
 	}, nil
 }
+
+func getRemoteFileBytes(address, filePath string, sshCfg *ssh.ClientConfig) ([]byte, error) {
+	conn, err := ssh.Dial("tcp", address, sshCfg)
+	if err != nil {
+		return nil, err
+	}
+	defer conn.Close()
+	c, err := sftp.NewClient(conn)
+	if err != nil {
+		return nil, err
+	}
+	defer c.Close()
+	r, err := c.Open(filePath)
+	if err != nil {
+		return nil, err
+	}
+	defer r.Close()
+	bytes, err := ioutil.ReadAll(r)
```

Comment:
Maybe we could simplify this with `return ioutil.ReadAll(r)`.

Reply:
Absolutely, that would be a good simplification.